spec/factories.rb

```diff
@@ -52,7 +52,9 @@
     end
 
     after(:build) do |organization, evaluator|
-      create_list(:user, evaluator.users_count, organizations: [organization])
+      if evaluator.users.count < evaluator.users_count
+        create_list(:user, evaluator.users_count, organizations: [organization])
```

Comment:
I'll try my best to explain it.

So this line of code will generate a list of users with length equals to `evaluator.users_count`, and the users are generated using [this factory](https://github.com/education/classroom/blob/master/spec/factories.rb#L74-L87). 

This is totally okay if we don't access any of the users generated by this line of code (and *yes*, there isn't a line in the codebase that does this). However, if we try to use any of the access tokens that are associated with these users to call GitHub APIs, it simply fails as these are just some random hex string.

This line of code will create a new user even if we already specified users to be associated with the organization (e.g. [GitHubFactory#classroom_org](https://github.com/education/classroom/blob/master/spec/support/github_factory.rb#L16-L26)).

This is the issue I faced when I run test examples:

```ruby
let(:organization) { classroom_org }
# => organization.users.size  <--- this is 2!
```

In this case, you can only get a working [organization.github_client](https://github.com/education/classroom/blob/master/app/models/organization.rb#L37:L40) on a 50% chance.

IMHO the reason we have this line of code is to ensure that there's at least one user in an organization, we don't need to add another user if we already got one. That's the reason I made this change.

Please let me know if there's anything I can help. Thanks! 😄